include/sound/sof/topology.h

```diff
@@ -28,7 +28,7 @@ enum sof_comp_type {
 	SOF_COMP_SRC,
 	SOF_COMP_SPLITTER,
 	SOF_COMP_TONE,
-	SOF_COMP_SWITCH,
+	SOF_COMP_DEPRECATED, // Formerly SOF_COMP_SWITCH
```

Comment:
I wonder if the same cannot be said of the SOF_COMP_SPLITTER 2 lines above? I don't recall we ever used those.

Reply:
Just grepped the repo, and checked `git log -S "SOF_COMP_SPLITTER" ` in SOF. It definitely looks unused. I will update to remove that as well